EDITORIAL CLOSING: Create Draft file for the XMPP Newsletter April 2022 (#1117)
xsf/xmpp.org: content/blog/2022-05-05-newsletter.md

- [Bifrost bridge: Offline messages exchange between Matrix and XMPP](https://opencollective.com/bifrost-mam)
- [Mellium Co-op](https://opencollective.com/mellium)

## Events

- [XMPP Office Hours](https://wiki.xmpp.org/web/XMPP_Office_Hours): available on our [YouTube channel](https://www.youtube.com/channel/UCf3Kq2ElJDFQhYDdjn18RuA)
- [Berlin XMPP Meetup (remote)](https://mov.im/?node/pubsub.movim.eu/berlin-xmpp-meetup): monthly meeting of XMPP enthusiasts in Berlin, every 2nd Wednesday of the month

## Articles

_Please do not simply drop links, rather try to make a short (<5 lines) and decent formulated description of what is the main outcome of your source. This will help us a lot! You do not need to use a hard, formal tone, be passionate about why this motivates you or what you work on!_

The [Mellium Dev Communiqué for March 2022](https://opencollective.com/mellium/updates/dev-communique-for-march-20222) has been released! This release includes changes to the sidebar in the Communiqué TUI client and improvements to various packages in the main mellium.im/xmpp module.

> 

Reviewer: ```suggestion

## XMPP Community Projects

Are you looking for an XMPP provider that suits you?
There is a [new website](https://providers.xmpp.net) based on the data of [XMPP Providers](https://invent.kde.org/melvo/xmpp-providers).
**XMPP Providers** has a curated list of providers and tools for filtering and creating badges for them.
The machine-readable list of providers can be [integrated in XMPP clients](https://providers.xmpp.net/apps/) to simplify the registration.
You can [help](https://invent.kde.org/melvo/xmpp-providers/-/blob/master/CONTRIBUTING.md) by improving your website (as a provider), by automating the manual tasks (as a developer), by adding new providers to the list (as an interested contributor).

```

Author: 49064b590a397ea4912f2129aeb18c0d01b109bf

## Software news

> 

Reviewer: ```suggestion

### Services

Are you looking for an XMPP provider that suits you?
There is a [new website](https://providers.xmpp.net) based on the data of [XMPP Providers](https://invent.kde.org/melvo/xmpp-providers).
**XMPP Providers** has a curated list of providers and tools for filtering them, creating badges for them and more.
The machine-readable list of providers can be [integrated in XMPP clients](https://providers.xmpp.net/apps/) to simplify the registration.

If you are a developer, please help us with automating the currently manual tasks.
If you are an operator of an already listed public provider, please update your website to improve your category.
You can see what is needed on your details page after selecting a provider entry and clicking the *Details* button on the [XMPP Providers website](https://providers.xmpp.net) .
If your provider is not listed yet, please [contribute to the list](https://invent.kde.org/melvo/xmpp-providers/-/blob/master/CONTRIBUTING.md).

```

Author: Ah wait please. This should not be more than 5 lines and also be named "XMPP Community Projects"

Reviewer: OK, I created https://github.com/xsf/xmpp.org/pull/1117/files#r856426227.

Author: 49064b590a397ea4912f2129aeb18c0d01b109bf

### Proposed

The XEP development process starts by writing up an idea and submitting it to the XMPP Editor. Within two weeks, the Council decides whether to accept this proposal as an Experimental XEP.

-   No XEPs proposed this month.

### New

-   Version 0.1.0 of [XEP-0462](https://xmpp.org/extensions/xep-0462.html) (PubSub Type Filtering)
    -   Accepted by vote of Council on 2022-02-09.

-   Version 0.1.0 of [XEP-0463](https://xmpp.org/extensions/xep-0463.html) (MUC Affiliations Versioning)
    -   Accepted by vote of Council on 2022-02-16.

> 

Reviewer 2: @Echolon Please check.

> 

Author: Your text is already in there. but I broke it down as the rule is one paragraph not more than ~5 lines of text.

- XMPP Newsletter via mail: We migrated to our [own mail-list server](https://mail.jabber.org/mailman/listinfo/newsletter) and stopped using Tinyletter. Its read-only and you will receive the XMPP Newsletter on a monthly basis. It also eliminates the privacy concerns with Tinyletter.
- By the way, have you checked our nice [XMPP RFC page](https://xmpp.org/rfcs/)? :-)

## XSF fiscal hosting projects

The XSF offers [fiscal hosting](https://xmpp.org/community/fiscalhost/) for XMPP projects. Please apply via [Open Collective](https://opencollective.com/xmpp). For more information, see the [announcement blog post](https://xmpp.org/2021/09/the-xsf-as-a-fiscal-host/). Current projects:

- [Bifrost bridge: Offline messages exchange between Matrix and XMPP](https://opencollective.com/bifrost-mam)
- NEW: [Mellium OMEMO encryption project](https://opencollective.com/mellium/projects/omemo)
- [Mellium Co-op](https://opencollective.com/mellium)

## XMPP Community Projects

A new community space for XMPP related projects and individuals has been created in the Fediverse! Join us on our new [Lemmy instance](https://community.xmpp.net/) and chat about all XMPP things! 

> Are you looking for an XMPP provider that suits you? There is a [new website](https://providers.xmpp.net) based on the data of [XMPP Providers](https://invent.kde.org/melvo/xmpp-providers). XMPP Providers has a curated list of providers and tools for filtering and creating badges for them. The machine-readable list of providers can be [integrated in XMPP clients](https://providers.xmpp.net/apps/) to simplify the registration. You can [help](https://invent.kde.org/melvo/xmpp-providers/-/blob/master/CONTRIBUTING.md) by improving your website (as a provider), by automating the manual tasks (as a developer), by adding new providers to the list (as an interested contributor). Read the [first blog post](https://providers.xmpp.net/blog/2022-04-25-start-of-xmpp-providers-website/)!

Reviewer 3: ```suggestion
Are you looking for an XMPP provider that suits you? There is a [new website](https://providers.xmpp.net) based on the data of [XMPP Providers](https://invent.kde.org/melvo/xmpp-providers). XMPP Providers has a curated list of providers and tools for filtering and creating badges for them. The machine-readable list of providers can be [integrated in XMPP clients](https://providers.xmpp.net/apps/) to simplify the registration. You can [help](https://invent.kde.org/melvo/xmpp-providers/-/blob/master/CONTRIBUTING.md) by improving your website (as a provider), by automating the manual tasks (as a developer), and by adding new providers to the list (as an interested contributor). Read the [first blog post](https://providers.xmpp.net/blog/2022-04-25-start-of-xmpp-providers-website/)!
```

Maxime “pep.” Buquet wrote some thoughts regarding _"Deal on Digital Markets Act: EU rules to ensure fair competition and more choice for users"_ in his [Interoperability in a “Big Tech” world](https://bouah.net/2022/04/interoperability-in-a-big-tech-world/) article. In a later article he describes [part of his threat model](https://bouah.net/2022/04/an-overview-of-my-threat-model/), detailing how XMPP comes into play and proposing ways it could be improved.

German "Freie Messenger" [shares some thoughts](https://www.freie-messenger.de/en/begriffe/interoperabilitaet/gedanken) on interoperability and the Digital Markets Act (DMA). They also offer [a comparison of "XMPP/Matrix"](https://www.freie-messenger.de/en/systemvergleich/xmpp-matrix)

## Software news

### Clients and applications

> [BeagleIM `5.2`](https://github.com/tigase/beagle-im/releases/tag/5.2) and [SiskinIM 7.2](https://github.com/tigase/siskin-im/releases/tag/7.2) just got released with fixes for OMEMO encrypted message in MUC channels, MUC participants disappearing randomly, and issues with VoIP call sending an incorrect payload during call negotiation.

Reviewer 3: ```suggestion
[BeagleIM `5.2`](https://github.com/tigase/beagle-im/releases/tag/5.2) and [SiskinIM `7.2`](https://github.com/tigase/siskin-im/releases/tag/7.2) just got released with fixes for OMEMO encrypted message in MUC channels, MUC participants disappearing randomly, and issues with VoIP call sending an incorrect payload during call negotiation.
```

> converse.js publishs version `9.1.0`. I comes with a new dark theme, several improvements of encryption (OMEMO), improvement of stanza timeout, font icons, updated translations and enhancement of the IndexedDB. Find the [release notes](https://github.com/conversejs/converse.js/releases/tag/v9.1.0).

Reviewer 3: ```suggestion
converse.js published version `9.1.0`. It comes with a new dark theme, several improvements for encryption (OMEMO), an improved stanza timeout, font icons, updated translations, and enhancements of the IndexedDB. Find more in the [release notes](https://github.com/conversejs/converse.js/releases/tag/v9.1.0).
```

JMP offers international call rates based on a [computing trie](https://blog.jmp.chat/b/2022-computing-call-rates). There are also [new commands and team members](https://blog.jmp.chat/b/april-newsletter-2022).

[Monal `5.1` has been released](https://github.com/monal-im/Monal/releases/tag/Build_iOS_780). This release brings OMEMO support in private group chats, communication notifications on iOS 15, and many improvements.

[PravApp project](https://prav.app/) is a plan to get a lot of people from India to invest small amounts to run an interoperable XMPP-based messaging service that is easier to join and discover contacts, similar to the Quicksy app. Prav will be Free Software, which respects users' freedom. The service will be backed by a cooperative society in India to ensure democratic decision making in which users can take part as well. Users will control the privacy policy of the service.

[Psi+ `1.5.1619` (2022-04-09)](https://sourceforge.net/projects/psiplus/files/Windows/Personal-Builds/tehnick/) has been released.

[Poezio `0.14`](https://bouah.net/2022/04/updates-from-the-poezio-ecosystem/) has been released alongside with multiple backend libraries. This new 

> release brings in lots of bug fixes and small improvements bu doesn't bring in big new features and shows that even if not perfect, poezio reached a point where devs are mostly happy about. Big changes are coming nonetheless, you will find out in the article. Stay tuned.

Reviewer 3: ```suggestion
release brings in lots of bug fixes and small improvements. Big changes are coming, read more in the article.
```

<!-- https://bouah.net/2022/04/poezio-full.png -->

[UWPX](https://uwpx.org) ships two small pre-release updates concering a [critical fix](https://github.com/UWPX/UWPX-Client/releases/tag/v.0.41.6.0) for a crash that occurs when trying to render an invalid user avatar and issues with the [Windows Store builds](https://github.com/UWPX/UWPX-Client/releases/tag/v.0.41.5.0). Besides that it also got a minor [UI update](https://github.com/UWPX/UWPX-Client/releases/tag/v.0.42.0.0) this month.

## Servers

Ignite Realtime Community:

> * Version `9.1.0` release 1 of the Openfire [inVerse plugin](https://discourse.igniterealtime.org/t/inverse-openfire-plugin-9-1-0-1-released/91560) was released which enables deployment of the third-party Converse client 37 in Openfire.

Reviewer 3: ```suggestion
- Version `9.1.0` release 1 of the Openfire [inVerse plugin](https://discourse.igniterealtime.org/t/inverse-openfire-plugin-9-1-0-1-released/91560) has been released, which enables deployment of the third-party Converse client 37 in Openfire.
```

> * Version `4.4.0` release 1 of the Openfire [JSXC plugin](https://discourse.igniterealtime.org/t/jsxc-openfire-plugin-4-4-0-1-released/91559) was released which enables deployment the third-party JSXC client 13 in Openfire.

Reviewer 3: ```suggestion
- Version `4.4.0` release 1 of the Openfire [JSXC plugin](https://discourse.igniterealtime.org/t/jsxc-openfire-plugin-4-4-0-1-released/91559) has been released, which enables deployment the third-party JSXC client 13 in Openfire.
```

> * Version `1.2.3` of the Openfire [Message of the Day plugin](https://discourse.igniterealtime.org/t/openfire-message-of-the-day-motd-plugin-version-1-2-3-released/91556) was released and it ships with the German translation to the admin console

Reviewer 3: ```suggestion
- Version `1.2.3` of the Openfire [Message of the Day plugin](https://discourse.igniterealtime.org/t/openfire-message-of-the-day-motd-plugin-version-1-2-3-released/91556) has been released, and it ships with German translations for the admin console
```

> * Version `1.8.0` of the Openfire [REST API plugin](https://discourse.igniterealtime.org/t/rest-api-openfire-plugin-1-8-0-released/91555/3) was released which adds new endpoints for readiness, liveliness and cluster status.

Reviewer 3: ```suggestion
- Version `1.8.0` of the Openfire [REST API plugin](https://discourse.igniterealtime.org/t/rest-api-openfire-plugin-1-8-0-released/91555/3) has been released, which adds new endpoints for readiness, liveliness and cluster status.
```

- Italian: [nicfab.it](https://www.nicfab.it/)
- Spanish: [xmpp.org](https://xmpp.org/categories/newsletter/)

## XSF Announcements

![XSF and Google Summer of Code 2022](/images/logos/GSoC_2022_Logo.png "XSF and Google Summer of Code 2022")

- The XSF has been accepted as hosting organization at [Google Summer of Code 2022 (GSoC)](https://xmpp.org/community/gsoc-2022/).
- XMPP Newsletter via mail: We migrated to our [own mail-list server](https://mail.jabber.org/mailman/listinfo/newsletter) and stopped using Tinyletter. Its read-only and you will receive the XMPP Newsletter on a monthly basis. It also eliminates the privacy concerns with Tinyletter.
- By the way, have you checked our nice [XMPP RFC page](https://xmpp.org/rfcs/)? :-)

## XSF fiscal hosting projects

The XSF offers [fiscal hosting](https://xmpp.org/community/fiscalhost/) for XMPP projects. Please apply via [Open Collective](https://opencollective.com/xmpp). For more information, see the [announcement blog post](https://xmpp.org/2021/09/the-xsf-as-a-fiscal-host/). Current projects:

- [Bifrost bridge: Offline messages exchange between Matrix and XMPP](https://opencollective.com/bifrost-mam)

> - [Collection Mellium Co-op projects](https://opencollective.com/mellium)

Reviewer 4: This can just be "Mellium Co-op", I don't know why it got changed, but this sounds weird. It's just one entity managed by the XSF.

## Events

- [XMPP Office Hours](https://wiki.xmpp.org/web/XMPP_Office_Hours): available on our [YouTube channel](https://www.youtube.com/channel/UCf3Kq2ElJDFQhYDdjn18RuA)
- [Berlin XMPP Meetup (remote)](https://mov.im/?node/pubsub.movim.eu/berlin-xmpp-meetup): monthly meeting of XMPP enthusiasts in Berlin, every 2nd Wednesday of the month

## Talks

Thilo Molitor presented their new even more [privacy friendly push design in Monal](https://xmpp-meetup.in-berlin.de/talks/monal-and-push.pdf) at the Berlin XMPP Meetup!

## Articles

> The Mellium Dev Communiqué for April 2022 has been released and can be found over on [Open Collective](https://opencollective.com/mellium/updates/dev-communique-for-april-2022). Furthermore, the [OMEMO Encryption](https://opencollective.com/mellium/projects/omemo) funding has reach $100!

Reviewer 4: I would remove the second sentence, it sounds odd to me and isn't an article anyways.

Author: Okay, was something you stated before in the other part

Reviewer 4: It made sense there, but I don't think it does here. If I wrote those exact words I apologize, it sounds weird, but maybe it's just the context.

Author: ok 9d9683c

- Version `1.2.3` of the Openfire [Message of the Day plugin](https://discourse.igniterealtime.org/t/openfire-message-of-the-day-motd-plugin-version-1-2-3-released/91556) has been released, and it ships with German translations for the admin console
- Version `1.8.0` of the Openfire [REST API plugin](https://discourse.igniterealtime.org/t/rest-api-openfire-plugin-1-8-0-released/91555/3) has been released, which adds new endpoints for readiness, liveliness and cluster status.

## Libraries

[slixmpp `1.8.2`](https://lab.louiz.org/poezio/slixmpp/-/releases/slix-1.8.2) has been released. It fixes RFC3920 sessions, improves certificate errors handling, and adds a plugin for XEP-0454 (OMEMO media sharing).

The mellium.im/xmpp library `v0.21.2` has been released! Highlights include support for [PEP Native Bookmarks](https://xmpp.org/extensions/xep-0402.html), and [entity capabilities](https://xmpp.org/extensions/xep-0115.html). For more information, see the [release announcement](https://opencollective.com/mellium/updates/new-release-mellium-im-xmpp-v0-21-2).

> The Mellium Dev Communiqué for April 2022 has also been released and can be found over on [Open Collective](https://opencollective.com/mellium/updates/dev-communique-for-april-2022). The big news this month is that the [OMEMO encryption project](https://opencollective.com/mellium/projects/omemo) has received donations totaling over $100!

Reviewer 4: This is a duplicate of the thing you moved to the articles section and can be removed.

    -   Reformulate to specify than only initial stanza and "unavailable" stanzas are transmitted with "presence" pemission.
    -   Namespace bump. (jp)

### Last Call

Last calls are issued once everyone seems satisfied with the current XEP status. After the Council decides whether the XEP seems ready, the XMPP Editor issues a Last Call for comments. The feedback gathered during the Last Call help improving the XEP before returning it to the Council for advancement to Stable.

-   No Last Call this month.

### Stable (formerly known as Draft)

Info: The XSF has decided to rename 'Draft' to 'Stable'. [Read more about it here](https://github.com/xsf/xeps/pull/1100).

-   No XEPs advanced to Stable this month.

### Deprecated

-   No XEP deprecated this month.

### Call for Experience

A Call For Experience - like a Last Call, is an explicit call for comments, but in this case it's mostly directed at people who've implemented, and ideally deployed, the specification. The Council then votes to move it to Final.

-   No Call for Experience this month.

## Spread the news!

> Please share the news on other networks:

Reviewer 4: Maybe add the Lemmy here too?

Author: Well, as you know that was my critic: Now its effectively official without any discussion and review.

And that way, I am not amused if this is the procedure to get things established...

9d9683c

Reviewer 4: None of the other places are official either, these are just social places where you can spread the news. There's nothing official about it.

Author: Yes, that was before I started in the team. I was simply asking to do things more aligned. But its in there now and I think we discussed already.

- The XSF has been accepted as hosting organization at [Google Summer of Code 2022 (GSoC)](https://xmpp.org/community/gsoc-2022/).
- XMPP Newsletter via mail: We migrated to our [own mail-list server](https://mail.jabber.org/mailman/listinfo/newsletter) and stopped using Tinyletter. Its read-only and you will receive the XMPP Newsletter on a monthly basis. It also eliminates the privacy concerns with Tinyletter.
- By the way, have you checked our nice [XMPP RFC page](https://xmpp.org/rfcs/)? :-)

## XSF fiscal hosting projects

The XSF offers [fiscal hosting](https://xmpp.org/community/fiscalhost/) for XMPP projects. Please apply via [Open Collective](https://opencollective.com/xmpp). For more information, see the [announcement blog post](https://xmpp.org/2021/09/the-xsf-as-a-fiscal-host/). Current projects:

- [Bifrost bridge: Offline messages exchange between Matrix and XMPP](https://opencollective.com/bifrost-mam)
- NEW: [Mellium OMEMO encryption project](https://opencollective.com/mellium/projects/omemo)
- [Mellium Co-op](https://opencollective.com/mellium)

## XMPP Community Projects

> A new community space for XMPP related projects and individuals has been created in the Fediverse! Join us on our new [Lemmy instance](https://community.xmpp.net/) and chat about all things XMPP! 

Reviewer: ```suggestion
A new community space for XMPP related projects and individuals has been created in the Fediverse! Join us on our new [Lemmy instance](https://community.xmpp.net/) and chat about all XMPP things! 
```

> Are you looking for an XMPP provider that suits you? There is a [new website](https://providers.xmpp.net) based on the data of [XMPP Providers](https://invent.kde.org/melvo/xmpp-providers). XMPP Providers has a curated list of providers and tools for filtering and creating badges for them. The machine-readable list of providers can be [integrated in XMPP clients](https://providers.xmpp.net/apps/) to simplify the registration. You can [help](https://invent.kde.org/melvo/xmpp-providers/-/blob/master/CONTRIBUTING.md) by improving your website (as a provider), by automating the manual tasks (as a developer), by adding new providers to the list (as an interested contributor). Read the [first blogpost](https://providers.xmpp.net/blog/2022-04-25-start-of-xmpp-providers-website/)!

Reviewer: ```suggestion
Are you looking for an XMPP provider that suits you? There is a [new website](https://providers.xmpp.net) based on the data of [XMPP Providers](https://invent.kde.org/melvo/xmpp-providers). XMPP Providers has a curated list of providers and tools for filtering and creating badges for them. The machine-readable list of providers can be [integrated in XMPP clients](https://providers.xmpp.net/apps/) to simplify the registration. You can [help](https://invent.kde.org/melvo/xmpp-providers/-/blob/master/CONTRIBUTING.md) by improving your website (as a provider), by automating the manual tasks (as a developer), by adding new providers to the list (as an interested contributor). Read the [first blog post](https://providers.xmpp.net/blog/2022-04-25-start-of-xmpp-providers-website/)!
```
